spec/ruby/core/time/shared/gm.rb

```diff
@@ -26,4 +26,28 @@
     time.usec.should == 0
     time.nsec.should == 999
   end
+
+  it "handles real leap seconds" do
+    with_timezone 'right/UTC' do
```

Comment:
Does a timezone have any effect on a Time created with Time.utc/Time.gm?
Also, does the timezone `right/UTC` exist?

Reply:
We're wading into territory I can't claim to really understand. But I derived the info from https://bugs.ruby-lang.org/issues/8885